docs/rfcs/XeGPU.md

```diff
@@ -132,14 +132,21 @@ When setting vnni_axis to 1, VNNI transformation has no impact on the physical d
   %at = XeGPU.load_nd %tdesc2 {vnni_axis = 1, mode = vc} :
      tile<8x16xbf16> into vector<8x8x2xbf16>
 ```
-VNNI transformation and transpose can be combined for low-precision data type like fp16, bf16, int8. The example below shows that a bf16 matrix [8row, 16col] is transposed to [16col, 8row], and then VNNI transform to [8col, 8row, 2col].  
-```mlir  
-  #sg_map_a = xegpu.sg_map<wi_layout = [2, 8], wi_data = [1, 2]>
-  %at = XeGPU.load_nd %block_a {transpose = [1, 0], vnni_axis = 0} :
-     tile<8x16xbf16, #sg_map_a> into vector<4x1x2bf16>
+VNNI transformation and transpose can not be combined. 
+
+Attribute `transpose_bit_width` specifies the bit_width of the data unit for the transpose during the load. The `transpose_bit_width` attribute overrides the element data type size for the transpose. For example, the transpose with "transpose_bit_width == 32` may applied to a tile with fp16 data type, which transposes the tile as if it is a tile of "fp16 pairs".  `transpose_bit_width` is vc mode only. 
+
+```mlir
+  %at = XeGPU.load_nd %tdesc1 {transpose = [1,0], transpose_bit_width = 32, mode = vc} :
```

Comment:
Did you omit sg_map_a intentionally?

Reply:
Yes. This advanced feature is for VC mode only.